src/assignment2/post.jsx

```diff
@@ -24,16 +25,67 @@ const usePost = (postName) => {
     return {post, isError, isLoading: (!isError && (post == null))};
 }
 
+const handleSubmit = async(event,newComment,postName) => {
+	event.preventDefault();
+	const formData = new FormData();
+	formData.append("new_comment", newComment);
+	const response = await fetch(`/post/${postName}/comments`, {
+	  method: "POST",
+	  body: formData
+	});
+	if (!response.ok) {
+		throw new Error(`HTTP error! status: ${response.status}`);
+	} 
+	console.log("successfull");
+	return newComment;
+};
+
+
+
 const Post = () => {
     const {postName} = useParams();
-    const {post, isLoading, isError} = usePost(postName);
+	const {post, isLoading, isError} = usePost(postName);
+	//const existingComment = useComments(postName);
+	const [newComment, setNewComment] = useState("");
+	const [existingComment, setExistingComment] = useState([]);
+	const [handleChange, setHandleChange] = useState("");
+	const count = 0;
+
+	useEffect(() => {
+		const comments = async(postName) => {
+			const resp = await fetch(`/post/${postName}/comments`);
+			const json = await resp.json();
+			const bufferArray = Buffer.from(json.postComments.data);
+			const lines = bufferArray.toString('utf8').split('\n');
+			console.log(lines);
+			setExistingComment(lines);
+		}
+		comments(postName);
+	}, [handleChange]);
+	//console.log("imageUrl: " + post);
+	//console.log("existingComments: " + existingComment);
+
+	function isNotEmptyComment(comment) {
+		return comment === '' ? false : true;
+	}
+
     return <>
 	       <Link to="/posts">Back</Link>
 	       {!!isLoading && <div>Loading...</div>}
 	       {!!isError && <div>Something went wrong</div>}
 	       {!isLoading && !isError && <ul>
 	       <h3>{postName}</h3>
 	       <img src={post} />
+		   <h3>Comments : </h3>
+		   <ul>
+	    		{existingComment.map(comment => isNotEmptyComment(comment) && <li>{comment}</li>)}
+	       </ul>
+		   <p>
+			<form onSubmit={e => setHandleChange(handleSubmit(e,newComment,postName))}>
```

Comment:
New comments are not showing up if I open a post which has comments in it. pls fix

https://user-images.githubusercontent.com/9821389/229825842-b5d47e5f-d904-4eb2-a248-02c7175554e4.mov